orangecontrib/timeseries/widgets/owtabletotimeseries.py

```diff
@@ -66,9 +66,10 @@ def set_data(self, data):
         if data.domain.has_continuous_attributes():
             vars = [var for var in data.domain.variables if isinstance(var, TimeVariable)] + \
                    [var for var in data.domain.variables
-                    if var.is_continuous and not isinstance(var, TimeVariable)]
+                    if var.is_continuous and not isinstance(var,
+                                                            TimeVariable)] + \
+                   [var for var in data.domain.metas if isinstance(var, TimeVariable)]
```

Comment:
Only accepts TimeVariables from metas? The two lines above list also other continuous variables. Like this:

```suggestion
                   [var for var in data.domain.metas if var.is_time] + \
                   [var for var in data.domain.metas if var.is_continuous]
```

Reply:
Yes, I wasn't sure whether to include numeric or not. Easy fix.

Reply:
Can I ask why does the first two checks use isinstance?`
`[var for var in data.domain.variables if isinstance(var, TimeVariable)]
Also, would it be nicer to:
[var for var in data.domain.metas if var.is_time or var.is_continuous]?
To avoid one extra loop?

Reply:
The reason for the extra loop was maybe that continuous and time variables become grouped. I'm actually :+1: for that.